CRM/Contract/Form/Cancel.php

```diff
@@ -52,6 +55,21 @@ function buildQuickForm () {
       [ "class" => "crm-select2" ]
     );
 
+    // Cancellation tags
+    $cancel_tags = (array) Api4\Tag::get(FALSE)
+      ->addWhere('parent_id:name', '=', 'contract_cancellation')
+      ->addSelect('name', 'label', 'description')
+      ->execute();
+
+    $this->add(
```

Comment:
I wonder if there's an easy way to make this input consistent with the tag input e.g. in the core activity form, which shows tag colors (if set):
![image](https://github.com/user-attachments/assets/553433ef-7b1c-498f-acd7-cb4f704bb4d4)

(Not important, so just if it's a quick fix.)